### DISCUSSION TITLE
Double SmartEVSE with double kWh-meters

### DISCUSSION THREAD

**Discussion by niekvanprooijen** [serkri/SmartEVSE-3#84] [Q&A]
Hi,

I currently have a "stupid" EV-box home charger and want to upgrade to SmartEVSE so I can apply load balancing and smart-charging when the price is low. We have 2 electric vehicles (Lynk&Co 01 PHEV capable of 7.4kW-1phase charging and a Nissan Leaf 2011 capable of 3.7kW-1phase charging), a 1x40A main connection with smart meter & P1 port which is connected to the synology NAS which serves HASS via a docker.
The EV-box is currently connected to a dedicated 16A breaker.
My plan is:
Change the 16A breaker for a 32A version and connect 2 SmartEVSE in a single box to this line. So:
Main cabinet:
1x40A main breaker --> P1meter --> 32A breaker --> Cable to EVSE box
EVSE box:
Modbus kWh meter --> SmartEVSE1 --> 2P contactor 32A -->  DC-leakage thingy --> fixed cable to car Lynk&Co
16A breaker --> Modbus kWh meter --> SmartEVSE2 --> 2P contactor 32A --> DC-leakage thingy --> fixed cable to car Nissan

Will this work? with load balancing, modbus communications, etc? I can send the P1 value from HASS to a smartEVSE via WiFi. Will this be picked up correctly and will loadbalancing work with 2 SmartEVSE's? Will the cars charge simultaniously or after each other?

Thank you in advance.

Best regards,
Niek

**Comment by dingo35** [comment]
Looking at your name you're probably Dutch (-speaking).

Post your question on tweakers.net, Forum "zelfbouw laadpalen ervaringen" and you will get loads of info and answers.

Short glance at your config: 16A breaker raises some questions (but the forum will have the answer), otherwise: yes it will work and it will charge both your cars at the same time, if enough current is available. 

niekvanprooijen ***@***.***> schreef op 6 januari 2023 13:54:28 CET:

**Reply by niekvanprooijen** [reply]
Thanks. I am Dutch :)
I actually just posted the same question there. But thanks anyway!
The 16A breaker is to protect the 16A charging cable to the Nissan. But if you are active on tweakers, let's discuss there.
